src/blockchain-utilities/derivation.ts

```diff
@@ -10,25 +10,19 @@ export const PROTOCOL_IDS = {
   AIRSEEKER: '5',
 };
 
-/**
- * An interface that reflects the structure of a Template
- */
-export interface Template {
-  airnode: string;
-  encodedParameters: string;
-  endpointId: string;
-}
-
 /**
  * Derives a template ID from the input parameters
  *
  * @param airnode an Airnode address
  * @param encodedParameters encoded parameters, see the airnode/abi package's encode function
  * @param endpointId An endpointID (see deriveEndpointId)
  */
-export const deriveTemplateId = ({ airnode, encodedParameters, endpointId }: Template) =>
+export const deriveTemplateIdV0 = (airnode: Address, encodedParameters: string, endpointId: string) =>
   ethers.utils.solidityKeccak256(['address', 'bytes32', 'bytes'], [airnode, endpointId, encodedParameters]);
 
+export const deriveTemplateIdV1 = (encodedParameters: string, endpointId: string) =>
```

Comment:
This is me being nit-picky but wouldn't it be nicer to have the args in the same order as they are used? (Disregard this if you feel I'm just being neurotic 🤭 )

Reply:
Good point. Not sure why I switched the order. I'd think I copied it from somewhere, but can't find the source anymore.

Reply:
Nvm, I see what I did wrong. I messed up, when I flattened the arguments from template object to arguments.